tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,379 @@
+- water selects bodies of water.
+
+
+The full list of features that you can modify:
+
+- all (default) selects all elements of the specified feature.
+- geometry selects all geometric elements of the specified feature.
+- geometry.fill selects only the fill of the feature's geometry.
+- geometry.stroke selects only the stroke of the feature's geometry.
+- labels selects the textual labels associated with the specified feature.
+- labels.icon selects only the icon displayed within the feature's label.
+- labels.text selects only the text of the label.
+- labels.text.fill selects only the fill of the label. The fill of a label is typically rendered as a colored outline that surrounds the label text.
+- labels.text.stroke selects only the stroke of the label's text.
+
+All styler options work with each of these features
+
+Let's consider a couple of examples to edit other elements and features
+
+```javascript
+
+//Remove icons
+var iconChange = [{ // Change map saturation.
+  "stylers": [{
+    "gamma": .2
+  }]
+}, { // Change label properties.
+  "elementType": "labels",
+  "stylers": [{
+    "visibility": "off",
+    "color": "#000055"
+  }]
+}, { // Change road properties.
+  "featureType": "road",
+  "elementType": "geometry",
+  "stylers": [{
+    "visibility": "off",
+    "color": "#000055"
+  }]
+}, { // Change road labels
+  "featureType": "road",
+  "elementType": "labels",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, { // Change icon properties
+  "elementType": "labels.icon",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, { // Change POI options
+  "featureType": "poi",
+  "elementType": "all",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, {
+  "featureType": "administrative",
+  "elementType": "geometry.fill",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, {
+  "featureType": "administrative",
+  "elementType": "geometry.stroke",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}];
+
+
+//Enhanced road network visualization
+
+var roadNetwork = [{
+    "stylers": [{
+      "saturation": -100
+  }]
+}, {
+    "featureType": "road.highway",
+    "elementType": "geometry.fill",
+    "stylers": [{
+      "color": "#000055"
+}, {
+    "weight": 2.5
+  }]
+}, {
+    "featureType": "road.highway",
+    "elementType": "geometry.stroke",
+    "stylers": [{
+      "color": "#000000"
+}, {
+    "weight": 2
+  }]
+}, {
+    "featureType": "road.arterial",
+    "elementType": "geometry",
+    "stylers": [{
+      "color": "#FF0000"
+}, {
+    "weight": 1.8
+  }]
+}, {
+    "featureType": "road.local",
+    "elementType": "geometry",
+    "stylers": [{
+      "color": "#00FF55"
+}, {
+    "weight": 1.5
+  }]
+}];
+
+Map.setOptions("roadNetwork", {
+  "iconChange": iconChange,
+  "roadNetwork": roadNetwork
+});
+```
+
+![Road](road.png)
+
+## Cheat codes
+
+There is a way to create almost any basemap you want without the hassle of tweaking any options. Enter https://snazzymaps.com/
```

Comment:
I think this will be a clickable hyperlink in GitHub's markdown, but please make it an explicit link (ditty for the other links on this page).

Reply:
Did this. Now, it just links to my patch-2 path though.